core/src/test/scala/unit/kafka/coordinator/transaction/TransactionStateManagerTest.scala

```diff
@@ -351,4 +389,16 @@ class TransactionStateManagerTest {
 
     EasyMock.replay(replicaManager)
   }
+
+  def writeTxnMarkersCallback(transactionalId: String,
```

Comment:
If we had a case class for the callback, this kind of thing would be much more concise